packages/components/src/input/checkbox.stories.tsx

```diff
@@ -41,6 +41,13 @@ storiesOf("check box", module)
   .add("slide check box", () => {
     return <TestSlideComponent />;
   })
-  .add("checkbox", () => {
-    return <TestComponent />;
-  });
+  .add("checkboxs", () => (
```

Comment:
typo

Reply:
Fixed, and made it a little prettier while I was at it.